Probesys/bileto
# imp: Improve the dashboard

## Related issue(s)



N/A

## Changes



- Add a "new ticket" link on the dashboard and global list of tickets
- Create a form to select an organization before creating a ticket
- Make the outline of the placeholders more visible
- Match “Your organizations” with singular on dashboard

## How to test manually



- with a user with permission "create tickets" for several organizations, go to the dashboard
- click on "New ticket" → check a form appears to select an orga and that you're redirected when submitting
- with a user with permission "create tickets" for only one organization, go to the dashboard
- click on "New ticket" → check that you're redirected to the "new ticket" form of your organization
- with a user with no permission "create tickets", go to the dashboard
- → check "New ticket" doesn't appear

## Checklist
